povm_toolbox/library/__init__.py

```diff
@@ -22,6 +22,7 @@
 
    POVMImplementation
    RandomizedProjectiveMeasurements
+   MutuallyUnbiasedBasesPOVM
```

Comment:
Is the suffix `POVM` really needed for this class name?
I am just asking because it looks a little bit inconsistent compared to the others which either end with `Shadows` or `Measurements`.

Reply:
Yes we should change this. I think simply removing `POVM` is a bit weird, so we could go for `MutuallyUnbiasedBasesMeasurements`. It might be a bit long tho, what do you think ?